config/config-builder/src/full_node_config.rs

```diff
@@ -247,13 +247,26 @@ impl FullNodeConfig {
     }
 
     fn build_seed_peers(&self, config: &NodeConfig) -> Result<SeedPeersConfig> {
-        let network = config
+        let seed_config = config
             .full_node_networks
             .last()
             .ok_or(Error::MissingFullNodeNetwork)?;
-        let mut seed_peers = HashMap::new();
-        seed_peers.insert(network.peer_id, vec![self.bootstrap.clone()]);
-        Ok(SeedPeersConfig { seed_peers })
+
+        let seed_handshake = 0;
+        let seed_pubkey = seed_config
+            .network_keypairs
+            .as_ref()
+            .ok_or(Error::MissingNetworkKeyPairs)?
+            .identity_keypair
+            .public_key();
+        let seed_base_addr = self.bootstrap.clone();
+        let seed_addr = seed_base_addr.append_prod_protos(seed_pubkey, seed_handshake);
+
+        let mut seed_peers = SeedPeersConfig::default();
+        seed_peers
+            .seed_peers
+            .insert(seed_config.peer_id, vec![seed_addr]);
```

Comment:
so we start with one peer that can have multiple addresses? Why not start with multiple peers?

Reply:
at least for the internal full node network, there's no reason not to have all peers in your seed config. only issue i think is config generation needs to be modified a bit (it's just taking a single seed peer address vs. all peer addresses). if we want to do this, we should do it in a separate pr